crates/bevy_gltf/src/loader.rs

```diff
@@ -1913,26 +1927,125 @@ impl AnisotropyExtension {
             .get("KHR_materials_anisotropy")?
             .as_object()?;
 
-        let (anisotropy_channel, anisotropy_texture) = extension
-            .get("anisotropyTexture")
-            .and_then(|value| value::from_value::<json::texture::Info>(value.clone()).ok())
-            .map(|json_info| {
-                (
-                    get_uv_channel(material, "anisotropy", json_info.tex_coord),
-                    texture_handle_from_info(load_context, document, &json_info),
-                )
-            })
-            .unzip();
+        let (anisotropy_channel, anisotropy_texture) = parse_material_extension_texture(
+            load_context,
+            document,
+            material,
+            extension,
+            "anisotropyTexture",
+            "anisotropy",
+        );
 
         Some(AnisotropyExtension {
             anisotropy_strength: extension.get("anisotropyStrength").and_then(Value::as_f64),
             anisotropy_rotation: extension.get("anisotropyRotation").and_then(Value::as_f64),
-            anisotropy_channel: anisotropy_channel.unwrap_or_default(),
+            anisotropy_channel,
             anisotropy_texture,
         })
     }
 }
 
+/// Parsed data from the `KHR_materials_specular` extension.
+///
+/// We currently don't parse `specularFactor` and `specularTexture`, since
+/// they're incompatible with Filament.
```

Comment:
Adding the note about specular vs reflectance-based PBR models from your PR description would be useful here.